### DISCUSSION TITLE
2023.05.30

### DISCUSSION THREAD

**Discussion by chwangmin** [Blue-club/pintos_5#11] [개발 일지]
# 오늘 할 일

김정인
- [ ] alarm-clock 블로그 업로드
- [x] priority scheduling 개념 이해 및 공부
- [x] priority scheduling and Synchronization
- [ ] Priority Inversion Problem

장혁
- [ ] 

최광민
- [ ] priority-sema, condvar PASS

**Comment by mywnajsldkf** [comment]
https://github.com/Blue-club/pintos_5/issues/10
